### DISCUSSION TITLE
password mismatch for created by pfa user

### DISCUSSION THREAD

**Discussion by jelebruh** [postfixadmin/postfixadmin#758] [Q&A] [answered]
Hi guys
I started server with postfix, dovecot, pfa and psql.

I added domain and user, send test email from web panel and got it. But when I tried login into imap from email clients - password mismatch.
In psql password exists, but md5 hash and in db hash not match.

Idk what I need to show from my configs, please specify - I show it.

**Comment by jelebruh** [comment, marked as answer]
ok im blind, nevermind. Forgot change encryption method

**Reply by DavidGoodwin** [reply]
Thanks for stating the solution

**Reply by jelebruh** [reply]
> Thanks for stating the solution


but one problem - `dovecot:` not working. This documented, but dovecot's methods those with caps (SHA512, MD5-CRYPT, ARGON2I)
There's why I'm stackd

@DavidGoodwin
